helm-charts/orchestrator/values.yaml

```diff
@@ -56,27 +56,13 @@ rhdhPlugins: # RHDH plugins required for the Orchestrator
   npmRegistry: "https://npm.registry.redhat.com" # NPM registry is defined already in the container, but sometimes the registry need to be modified to use different versions of the plugin, for example: staging(https://npm.stage.registry.redhat.com) or development repositories
```

Comment:
@jenniferubah the -rc plugin versions are only available in the staging registry

Reply:
@batzionb, this is a default value and can be overridden via the CR. And QE can configure it for their testing.

Reply:
we want to have none functional values in our main branch?

Reply:
@masayag WDYT?

Reply:
> we want to have none functional default values in our main branch?

If it's from a customer's perspective, I don't think they would have access to the staging registry right?
I had not changed during the last release and QE was able to test with it. So that's why I didn't need to change it as we prep for 1-4.

@jordigilh - do we usually change this while changing plugin version? I can change it in the next PR, if so.

Reply:
We must have https://github.com/rhdhorchestrator/orchestrator-helm-operator/pull/377  (bump RHDH to 1.4) before merging this PR, since only in RHDH-1.4 the notifications plugins are in-image.

as for Orchestrator plugins and NPM registry - we should maintain their compatibility and make sure that each change to the plugins matches the default npm registry (when release, it will be updated to https://npm.registry.redhat.com), but prior to it, it can only be installed internally. 

it isn't optimal since we won't be able to install it on demo clusters, unless we publish the plugins externally.

Reply:
Does it matter whether this PR is merged before or after? This branch (main) targets release-1.4/RHDH 1.4 anyway. When we change to reference the RHDH 1.4 operator channel should not deter us from merging any changes as long as they don't overlap or have dependencies to each other. What we should know is that the code is not ready to be tested until the other PR is merged to reflect the change in dependency versions.

 Or maybe I'm missing something...?

Reply:
If this PR merged before, it will be broken, since in this PR we're still using RHDH 1.3.
Since this PR contains changes that were introduced only in RHDH-1.4, we need the other PR to be merged.
So with the other PR, we'll maintain integrity between PRs while with this one, this branch will be broken and will prevent from other developers to send/test PRs.

Reply:
Merged [RHDH 1.4](https://github.com/rhdhorchestrator/orchestrator-helm-operator/pull/437) changes first, before this one.